examples/cpp/LaunchCartPole.cpp

```diff
@@ -150,7 +155,7 @@ int main(int argc, char* argv[])
         // Handle termination
         if (state->done) {
             gymppDebug << "The environment reached the terminal state" << std::endl;
-            std::this_thread::sleep_for(std::chrono::milliseconds(500));
+            std::this_thread::sleep_for(std::chrono::milliseconds(50));
```

Comment:
It was already there, but why there is this sleep? Just as an example or otherwise there are problems of  some kind?

Reply:
It is just for visualization purposes. If there is no sleep, it is very difficult to understand when an episode ends. Since this file is an example it can stay as it is, for serious code it should be not there.